lib/tasks/permissions.rake

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+namespace :permissions do
+  desc 'Generate YAML from CSV'
```

Comment:
Temporary command until we settle on the final permission definitions.
The CSV file won't make it to the final version either.